tests/test_api.py

```diff
@@ -963,7 +963,7 @@ def test_get_item_not_found(self):
         with model1_data(self.appbuilder.session, 1):
             model_id = MODEL1_DATA_SIZE + 1
             rv = self.auth_client_get(client, token, f"api/v1/model1api/{model_id}")
-            self.assertEqual(rv.status_code, 404)
+            self.assertEqual(rv.status_code, 200)
```

Comment:
strange why this has changed? can you check

Reply:
@dpgaspar : he changed it, because intermittently `model_id`  seems to conflict with auto incremented database keys. I'd like to propose the change `model_id = 0` instead.